src/core/hle/service/hid/controllers/npad.cpp

```diff
@@ -338,6 +340,25 @@ void Controller_NPad::OnUpdate(const Core::Timing::CoreTiming& core_timing, u8*
             cur_entry.timestamp2 = cur_entry.timestamp;
         }
 
+        const std::array<SixAxisGeneric*, 6> controller_sixaxes{&npad.full,       &npad._handheld,
+                                                                &npad._left_dual, &npad._right_dual,
+                                                                &npad._left,      &npad._right};
```

Comment:
Adding a comma at the end of the list, should allow clang-format to format this more nicely.

```suggestion
                                                                &npad._left,      &npad._right,};
```